packages/charting/src/components/Legends/Legends.styles.ts

```diff
@@ -20,7 +21,7 @@ export const getStyles = (props: ILegendStyleProps): ILegendsStyles => {
     },
     rect: {
       selectors: {
-        '@media screen and (-ms-high-contrast: active)': {
+        [HighContrastSelector]: {
```

Comment:
[HighContrastSelector definition](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/styles/CommonStyles.ts) for reference (it's identical to this)